packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout-selectors.js

```diff
@@ -54,5 +54,9 @@ export const FIELDS = {
     userNote: {
         input: '[data-test-id="formfield-textarea"]',
         error: ''
+    },
+    tableIdentifier: {
+        input: '[data-test-id="formfield-table-identifier-input"]',
+        maxLengthError: '[data-test-id="error-table-identifier-max-length"'
```

Comment:
Missing close square bracket?
```suggestion
        maxLengthError: '[data-test-id="error-table-identifier-max-length"]'
```